migrations/V148__System_Intake_Edits_Requested_Add_Step_To_Action.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE actions ADD COLUMN step system_intake_step;
```

Comment:
This file should end with a newline; I'm not sure why the "Add newlines to ends of file" lint didn't catch this in CI, but when I added this locally, the pre-commit hook that runs that lint failed and added a newline.

Reply:
I tried messing with this and it passes the linter for me as is, and removes an extra line if I add one.